units/src/parse.rs

```diff
@@ -252,8 +253,8 @@ pub fn hex_u128(s: &str) -> Result<u128, ParseIntError> {
 ///
 /// # Errors
 ///
-/// - If input string does not contain a `0x` (or `0X`) prefix.
-/// - If input string is not a valid hex encoding of a `u128`.
+/// - If the input string does not contain a `0x` (or `0X`) prefix.
+/// - If the input string is not a valid hex encoding of a `u128`.
```

Comment:
I'm fine with this change but I think the original was correct too. Mentioning it because we discussed this before because it is hard  for English-second-language speakers to know which is correct.